web/server.js

```diff
@@ -225,6 +260,19 @@ module.exports = function(options) {
           {
             assign: 'authCode',
             method: function(request, reply) {
+              if ( request.pre.grant_type === 'password' ) {
+                return account.verifyPassword(request, function(err, json) {
+                  if ( err ) {
+                    return reply(err);
+                  }
+
+                  reply({
+                    user_id: json.user.username,
+                    scopes: request.payload.scopes
```

Comment:
we could ask for clients to declare scopes in advance for the password grant. I just followed what the spec said to do.